src/util/sienna/helper_functions.jl

```diff
@@ -79,11 +79,19 @@ function line_rating(line::PSY.Branch)
     error("line_rating isn't defined for $(typeof(line))")
 end
 
-function line_rating(line::Union{PSY.Line, PSY.MonitoredLine})
+function line_rating(line::PSY.Line)
     rate = PSY.get_rating(line)
     return (forward_capacity=abs(rate), backward_capacity=abs(rate))
 end
 
+function line_rating(line::PSY.MonitoredLine)
+    rate = PSY.get_flow_limits(line)
+    return (
+        forward_capacity=abs(getfield(rate, :from_to)),
```

Comment:
Use `.from_to` and `.to_from`. Also do you know the sign convention on these values?

Reply:
The from_to and to_from part takes care of sign convention. I believe the actual values should always be positive.

Reply:
Then we don't need abs, do we?